docs/docs-site/content/administrator/configuration/connectors/_index.md

```diff
@@ -758,7 +758,7 @@ Alternative:
 
 ### Dask SQL
 
-[dask-sql](https://dask-sql.readthedocs.io) adds a SQL layer on top of Dask.
+dask-sql adds a SQL layer on top of Dask.
```

Comment:
can we use this link ->`https://dask-sql.readthedocs.io/en/latest/` ?

Reply:
Its giving 403 error